src/templates/plugin/boilerplate/index.js.ejs.ejs

```diff
@@ -16,5 +16,5 @@
 import { AppRegistry } from 'react-native'
 import App from './App/App.js'
 
-AppRegistry.registerComponent('<%= props.name %>', () => App)
+AppRegistry.registerComponent('<%- '<' + '%= props.name %' + '>' %>', () => App)
```

Comment:
Need to retain the <%= props.name %> after we generate this, so have to break it up like this.